website/settings/defaults.py

```diff
@@ -122,7 +122,11 @@ def parent_dir(path):
 
 USE_EMAIL = True
 FROM_EMAIL = 'openscienceframework-noreply@osf.io'
-SUPPORT_EMAIL = 'support@osf.io'
+
+# support email
+OSF_SUPPORT_EMAIL = 'support@osf.io'
```

Comment:
After this is launched, I think we'll make sure all of the environments are up-to-date with an override for this, then do a follow-up ticket that makes the default something else. In the meantime, let's add to the local-dist.py fake-support@osf/cos.io for these two variables.